api/test/queries.test.js

```diff
@@ -489,19 +489,130 @@ describe('queries', () => {
               query: `{
                  dwellings:dwellingsInFSA(
                   forwardSortationArea: "C1A"
-                  filter: {field: city eq: "Charlottetown"}
+                  filter: {field: dwellingCity eq: "Charlottetown"}
+                 ) {
+                   results {
+                     city
+                   }
+                 }
+               }`,
+            })
+          let { dwellings: { results: [first] } } = response.body.data
+          expect(first.city).toEqual('Charlottetown')
+        })
+      })
+
+      describe('lt: less than', () => {
+        it('filters out results where the selected field has a value less than the selected value', async () => {
```

Comment:
Pretty nitpicky, but I would interpret 'filtering out' a result as removing it from the results.

Ie, if `results = [1,2,3]` and we only want results less than 2, I would say that [2,3] have been filtered out. (ie, `results = [1]`)

Maybe you could say "it keeps/selects results where the selected field...".